x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -189,6 +194,11 @@ export function ServiceInventory() {
             <MLCallout onDismiss={() => setUserHasDismissedCallout(true)} />
           </EuiFlexItem>
         )}
+        {!searchAggregatedTransactions && (
+          <EuiFlexItem>
+            <AggregatedTransactionsCallout />
+          </EuiFlexItem>
```

Comment:
Can we add this to one of the templates to avoid duplication?

Reply:
if we added it to the template, then it would be visible everywhere, but we might only want to show the callout where we make this automatic decision: service inventory, service overview, transaction overview, and trace overview. Service maps i think we can safely omit since it has a hard dependency on transaction events to build the map, and dependencies has a hard dependency on metrics.

Reply:
Okay, makes sense 👍